client/src/components/AuditAdmin/Setup/Contests/ContestForm.tsx

```diff
@@ -381,26 +360,21 @@ const ContestForm: React.FC<IProps> = ({
               )}
             />
           </FormWrapper>
-          {nextStage.state === 'processing' ? (
-            <Spinner />
-          ) : (
```

Comment:
Is some other loading indicator replacing this and other spinners removed in this commit? Or is a loading indicator no longer necessary? (My simplistic search for "Spinner" in this commit didn't surface any replacements)

Reply:
I couldn't figure out what this spinner was actually needed for. The submit button has a spinner using the `loading` property, so that seemed sufficient to me